qos/evm/response_generic.go

```diff
@@ -0,0 +1,46 @@
+package evm
+
+import (
+	"encoding/json"
+
+	"github.com/buildwithgrove/path/qos/jsonrpc"
+)
+
+// responseUnmarshallerGeneric unmarshal the provided byte slice
+// into a responseGeneric struct and saves any data that may be
+// needed for producing a response payload into the struct.
+func responseUnmarshallerGeneric(data []byte) (response, error) {
+	var response responseGeneric
+	err := json.Unmarshal(data, &response)
+	if err != nil {
+		// TODO_INCOMPLETE: implement a method-specific validator of the response.
```

Comment:
Please put your name on incomplete stuff like this

Reply:
Sure, this particular instance is more of a `TODO_FUTURE` though. Comment updated.